chaste_codegen/chaste_model.py

```diff
@@ -279,12 +291,22 @@ def _get_membrane_voltage_var(self):
         """ Find the membrane_voltage variable"""
         voltage = self._model.get_variable_by_ontology_term((self._OXMETA, "membrane_voltage"))
         desired_units = self._units.get_unit('millivolt')
+        # If V is not a state var add annotation as modifiable parameter or derived quantity as appropriate.
+        if voltage not in self._state_vars:
```

Comment:
Could extract the logic here as a method so it can be called for calcium below too?

Reply:
yeah good point I'll do that